### PR TITLE
Amend HGV/TRL groups 1 and 4

### PR DESCRIPTION
**Pull request by leifkemp-bjss**
## CB2-6535 and CB2-6537: Amend HGV/TRL groups 1 and 4

_Templates for amending HGV/TRL groups 1 and 4_
[CB2-6535](https://dvsa.atlassian.net/browse/CB2-6535)
[CB2-6537](https://dvsa.atlassian.net/browse/CB2-6537)

## Checklist

- [ ] Branch is rebased against the latest develop/common
- [ ] Necessary `id` required prepended with `"test-"` have been checked with automation testers and added
- [ ] Code and UI has been tested manually after the additional changes
- [ ] PR title includes the JIRA ticket number
- [ ] Squashed commits contain the JIRA ticket number
- [ ] Link to the PR added to the repo
- [ ] Delete branch after merge

### CODE REVIEW

File: src/app/forms/templates/test-records/master.template.ts [hunk 1]
```diff
@@ -261,6 +264,26 @@ export const masterTpl: Record<VehicleTypes, Partial<Record<keyof typeof TEST_TY
       reasonForCreation: reasonForCreationSection,
       required: SpecialistRequiredSectionHGVTRL,
       customDefects: CustomDefectsSection
+    },
+    testTypesDeskBasedGroup1: {
+      required: RequiredSection,
+      vehicle: DeskBasedVehicleSectionHgvGroup124,
```
Comment (tom-bjss) [inline src/app/forms/templates/test-records/master.template.ts:270]: Whats 124 like 1 to 4 ?

Comment (leifkemp-bjss) [reply src/app/forms/templates/test-records/master.template.ts:270]: Groups 1, 2 and 4

**Review by HazelWilderspin — APPROVED** [review]

File: src/app/forms/templates/test-records/master.template.ts (hunk 1 above)
Comment (gjulien-bjss) [reply src/app/forms/templates/test-records/master.template.ts:273]: should it be name 1And2And4 then? keep it consistent with other templates?

File: src/app/forms/templates/test-records/section-templates/test/desk-based/desk-based-test-section-group4-PSV.template.ts [hunk 2]
```diff
@@ -143,6 +143,16 @@ export const amendDeskBasedTestSectionGroup4Psv: FormNode = {
           name: '0', // it is important here that the name of the node for an ARRAY type should be an index value
           type: FormNodeTypes.GROUP,
           children: [
+            {
```
Comment (mcadenas-bjss) [inline src/app/forms/templates/test-records/section-templates/test/desk-based/desk-based-test-section-group4-PSV.template.ts:146]: I think this should be in the required section which I've added in my PR

**Review by mcadenas-bjss — DISMISSED** [review]

**Review by HazelWilderspin — APPROVED** [review]

**Review by aaron-downing — APPROVED** [review]

**Review by Ian-Hodges — APPROVED** [review]

**Pull request merged**